src/components/StartBlock.tsx

```diff
@@ -37,7 +37,8 @@ const futureBox = classnames(
   flexDirection('md:flex-row', 'flex-col-reverse'),
   alignItems('items-center'),
   justifyContent('justify-end'),
-  space('md:space-x-4')
+  space('md:space-x-4'),
+  margin('mt-32', 'sm:mt-40', 'lg:mt-0')
```

Comment:
I don't think we want to use too big of margins hmmm, would using space work as an alternative solutions?

Reply:
Let me try that out

Reply:
This is a large margin. Why do we need it?